cognite/powerops/preprocessor/cogreader.py

```diff
@@ -18,6 +19,33 @@
 logger = logging.getLogger(__name__)
 
 
+class CogShopFile(BaseModel):
+    label: Optional[str]
+    pick: Literal["closest", "latest", "all"]
+    sort_by: Optional[str]
+    external_id_prefix: Optional[str]
+    external_id: Optional[str]
```

Comment:
Do you need both of these, they seem redundant? Isn't `external_id` built up of the other fields? And thus could be a property?

Reply:
I agree, but maybe for transparency it's clearer if the users puts in external_id in the config instead of just the prefix?